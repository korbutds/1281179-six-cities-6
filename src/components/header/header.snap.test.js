import React from 'react';
import {render} from '@testing-library/react';
import {Test} from '../../test/test-mock';
import Header from './header';
test(`Header component render correctly`, () => {
  const {container} = render(
      <Test>
        <Header />
      </Test>
  );
  expect(container).toMatchSnapshot();
});
