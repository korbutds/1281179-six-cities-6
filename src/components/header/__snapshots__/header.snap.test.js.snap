// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component render correctly 1`] = `
<div>
  <header
    class="header"
  >
    <div
      class="container"
    >
      <div
        class="header__wrapper"
      >
        <div
          class="header__left"
        >
          <a
            class="header__logo-link "
            href="/"
          >
            <img
              alt="6 cities logo"
              class="header__logo"
              height="41"
              src="img/logo.svg"
              width="81"
            />
          </a>
        </div>
        <nav
          class="header__nav"
        >
          <ul
            class="header__nav-list"
          >
            <li
              class="header__nav-item user"
            >
              <a
                class="header__nav-link header__nav-link--profile"
                href="/favorites"
              >
                <div
                  class="header__avatar-wrapper user__avatar-wrapper"
                  style="background-image: url(https://www.avatar.ru); border-radius: 50%;"
                />
                <span
                  class="header__user-name user__name"
                >
                  Fake login
                </span>
              </a>
            </li>
            <li
              class="header__nav-item"
            >
              <a
                class="header__nav-link header__nav-link--profile"
                href="/"
                style="margin-left: 10px;"
              >
                <span
                  class="header__user-name user__name"
                >
                  Выход
                </span>
              </a>
            </li>
          </ul>
        </nav>
      </div>
    </div>
  </header>
</div>
`;
